src/App.js

```diff
@@ -23,13 +27,25 @@ function App() {
       //Reset states on new zipcode
       setHoverEvent(null);
       setLocFilt(null);
+      setCardIndex(0);
 
     }
   }, [currZip]);
 
+  //Card index utilizes the hoverEvent to highlight the card's respective marker
+  useEffect(() => {
+    if(isMobile && events != null){
+      setHoverEvent((('location' in events[cardIndex] && 'location' in events[cardIndex]['location'] && 'latitude' in events[cardIndex]['location']['location']) ? "" + events[cardIndex]['location']['location']['latitude'] + "&" + events[cardIndex]['location']['location']['longitude'] : null));
```

Comment:
Nit: this line is silly long. Maybe consider a linter and/or auto-formatter to take care of that stuff